eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -2384,6 +2385,62 @@ public function testCopySubtreeWithAliases()
         $this->assertGeneratedAliases($urlAliasService, $expectedSubItemAliases);
     }
 
+    /**
+     * @see \eZ\Publish\API\Repository\LocationService::copySubtree()
```

Comment:
This is better:
```suggestion
     * @covers \eZ\Publish\API\Repository\LocationService::copySubtree
```
Though @Steveb-p made recently some remarks about `@covers`, that it should be placed on top of the class
`@covers \eZ\Publish\API\Repository\LocationService`. Feel free to replace `@see` there as it does not provide much value there.

Reply:
All remarks resolved in https://github.com/ezsystems/ezpublish-kernel/pull/3127/commits/506073b896532feea4e858544130a41e8e399250